packages/interactive_media_ads/lib/src/platform_interface/platform_ads_manager.dart

```diff
@@ -20,7 +18,7 @@ abstract class PlatformAdsManager {
   PlatformAdsManager();
 
   /// Initializes the ad experience using default rendering settings.
-  Future<void> init(AdsManagerInitParams params);
+  Future<void> init([PlatformAdsRenderingSettings? settings]);
```

Comment:
This should be a named parameter unless we're very sure there will never be any other init params.